src/main/java/practice/StreamPractice.java

```diff
@@ -1,80 +1,80 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
-    /**
-     * Given list of strings where each element contains 1+ numbers:
-     * input = {"5,30,100", "0,22,7", ...}
-     * return min integer value. One more thing - we're interested in even numbers.
-     * If there is no needed data throw RuntimeException with message
-     * "Can't get min value from list: < Here is our input 'numbers' >"
-     */
-    public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+    public static int findMinEvenNumber(List<String> numbers) {
+        OptionalInt n = numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(",")))
+                .map(Integer::valueOf)
+                .filter(i -> i % 2 == 0)
+                .mapToInt(i -> i)
+                .min();
+        return n.orElseThrow(() -> new RuntimeException("Can't get min value from list: "
+                + "< Here is our input 'numbers' >"));
```

Comment:
do not use abbreviations, fix everywhere
```suggestion
        return numbers.stream()
                .flatMap(lineNumbers -> Arrays.stream(lineNumbers.split(",")))
                .mapToInt(Integer:: parseInt)
                .filter(number -> number % 2 == 0)
                .min()
                .orElseThrow(() -> new RuntimeException("Can't get min value from list: " + numbers));
```